src/components/ErrorModal.vue

```diff
@@ -0,0 +1,267 @@
+<template>
+  <div
+    class="error-modal"
+    @click="resolve"
+  >
+    <div class="modal-body" @click.stop>
+      <IconClose
+        class="close-modal"
+        @click="resolve"
+      />
+      <template v-if="step === 1">
+        <FailureIcon />
+        <h1>{{ $t('reportBug.Title') }}</h1>
+        <h2>{{ $t('reportBug.SubTitle') }}</h2>
+        <p class="description">
+          {{ $t('reportBug.Description') }}
+        </p>
+        <div
```

Comment:
```suggestion
        <button
```
or
```suggestion
        <ButtonPlain
```

Reply:
changed, but it's for other line

Reply:
https://github.com/aeternity/superhero-ui/blob/5904e141e1ffb4d6b751d74f8df6545acd3174ac/src/components/ErrorModal.vue#L18